samila/functions.py

```diff
@@ -38,6 +38,16 @@ def random_equation_gen():
     return result
 
 
+def random_hex_color_gen():
+    """
+    Generate random hex color code.
+
+    :return: color code as str
+    """
+    random_color = "#%06x" % random.randint(0, 0xFFFFFF)
```

Comment:
`0` and `0xFFFFFF` are both inclusive?

Reply:
Yeah.
Take a look at this [link](https://docs.python.org/3.6/library/random.html#random.randint)